### ISSUE TITLE
Highlight selected subtitle version

### ISSUE THREAD

**Issue by oas777** [polimediaupv/paella-ethz#41]
... but I found another small issue: The subtitles selected (or disabled) should be highlighted so users know what they have chosen.
![grafik](https://user-images.githubusercontent.com/1752342/236638131-a8099ef3-465f-4313-90b2-2a8f27c416fa.png)

**Comment by miesgre** [comment]
Fixed!
Demo: https://polimediaupv.github.io/paella-ethz/?id=belmar_16_9_hls
